cmake/modules/SwiftUtils.cmake

```diff
@@ -3,3 +3,12 @@ function(precondition var)
     message(FATAL_ERROR "Error! Variable ${var} is false or not set!")
   endif()
 endfunction()
+
+macro(list_replace lst old new)
```

Comment:
Yeah, I agree, functions are better as they don't pollute the global namespace.  Ill see if I can get that changed.